indico/util/translations.py

```diff
@@ -17,11 +17,26 @@
 
 from functools import wraps
 
-from indico.util.string import encode_if_unicode
+from jinja2.ext import babel_extract
+
+from indico.util.string import encode_if_unicode, trim_inner_whitespace
 
 
 def ensure_str(fn):
     @wraps(fn)
     def wrapper(*args, **kwargs):
         return encode_if_unicode(fn(*args, **kwargs))
     return wrapper
+
+
+# TODO: Remove this once there's proper support in upstream Jinja
+# https://github.com/pallets/jinja/pull/683
+def jinja2_babel_extract(fileobj, keywords, comment_tags, options):
+    """
+    Custom babel_extract for Jinja.
```

Comment:
I really don't like the asymmetry created by the lack of newline. But anyway, it's true that that's the style we mostly use.